.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/HazyResearch/flash-attention.git
 [submodule "third_party/cutlass"]
 	path = third_party/cutlass
-	url = https://github.com/NVIDIA/cutlass.git
+	url = https://github.com/hwu36/cutlass.git
```

Comment:
Is this change still needed? Looks like everything has been pushed to NVIDIA repo already?